tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932260.yaml

```
@@ -380,3 +380,35 @@ tests:
             version: HTTP/1.0
           output:
             no_log_contains: id "932260"
+  - test_title: 932260-24
+    desc: "FP with word perl Github Issue #3490"
```

Do not reference GitHub issues please. If we should ever change platforms in the future, that information would become useless.
```suggestion
    desc: "FP against name 'Perlak' (matches 'perl')"
```